packages/plugins/typescript/typescript/src/visitor.ts

```diff
@@ -88,8 +88,14 @@ export class TsVisitor<TRawConfig extends TypeScriptPluginConfig = TypeScriptPlu
     const typeString = this.config.wrapFieldDefinitions ? `FieldWrapper<${node.type}>` : ((node.type as any) as string);
     const originalFieldNode = parent[key] as FieldDefinitionNode;
     const addOptionalSign = !this.config.avoidOptionals.object && originalFieldNode.type.kind !== Kind.NON_NULL_TYPE;
-    const comment = transformComment((node.description as any) as string, 1);
     const { type } = this.config.declarationKind;
+    let commentText: string = node.description as any;
```

Comment:
@adelsz I think this could be moved to the Base visitor? this way it will be applied to `flow` plugins as well, and not only `typescript`.